src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -817,5 +817,10 @@ public void ShowTaskbarIcon(bool value)
                 UnmanagedMethods.ShowWindow(_hwnd, windowPlacement.ShowCmd);
             }
         }
+
+        public void SetOwner(IWindowImpl owner)
+        {
+            UnmanagedMethods.SetWindowLong(_hwnd, -8, (uint)owner.Handle.Handle);
```

Comment:
Do not do this: https://msdn.microsoft.com/en-us/library/windows/desktop/ms633591(v=vs.85).aspx

>You must not call SetWindowLong with the GWL_HWNDPARENT index to change the parent of a child window. Instead, use the SetParent function.

Reply:
as discussed on Gitter GTK and WPF do this see comments here:
https://github.com/GNOME/gtk/blob/ae7a6773468181b3524c12426f70749682ff0fce/gdk/win32/gdksurface-win32.c#L1800

and also:
https://referencesource.microsoft.com/#PresentationFramework/src/Framework/System/Windows/Window.cs,4257

Please allow me to confirm I hadn't looked at reference source before implementing this, and have only looked now to ensure our compatibility with windows / wpf.